lib/Deque.js

```diff
@@ -13,10 +13,10 @@ class Deque {
 
   /**
    * removes and returns the first element from the queue
-   * @return {[type]} [description]
+   * @return {any} [description]
    */
   shift() {
-    if (this._length === 0) {
+    if (this.length === 0) {
```

Comment:
Here, `this._length` points to nothing, while other methods use `this.length` that is a getter for `this._list.length`